presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -0,0 +1,228 @@
+=================
+Dynamic Filtering
+=================
+
+Dynamic filtering optimizations significantly improve the performance of queries
+with selective joins by avoiding reading of data that would be filtered by join condition.
+
+Consider the following query which captures a common pattern of a fact table ``store_sales``
+joined with a filtered dimension table ``date_dim``:
+
+    SELECT COUNT(*) FROM
```

Comment:
not sure if we should format it across lines differently..

```
SELECT count(*) 
FROM store_sales
JOIN date_dim ON store_sales.ss_sold_date_sk = date_dim.d_date_sk
WHERE d_following_holiday='Y' 
AND d_year = 2000;
```
or so?

Reply:
re-formatted